be/src/vec/exec/vanalytic_eval_node.cpp

```diff
@@ -203,15 +203,17 @@ Status VAnalyticEvalNode::prepare(RuntimeState* state) {
     }
     _fn_place_ptr = _agg_arena_pool->aligned_alloc(_total_size_of_aggregate_states,
                                                    _align_aggregate_states);
-    RETURN_IF_CATCH_EXCEPTION(static_cast<void>(_create_agg_status()));
-    _executor.insert_result =
-            std::bind<void>(&VAnalyticEvalNode::_insert_result_info, this, std::placeholders::_1);
-    _executor.execute =
-            std::bind<void>(&VAnalyticEvalNode::_execute_for_win_func, this, std::placeholders::_1,
-                            std::placeholders::_2, std::placeholders::_3, std::placeholders::_4);
+    RETURN_IF_CATCH_EXCEPTION(THROW_IF_ERROR(_create_agg_status()));
+    _executor.insert_result = [this](auto&& PH1) {
```

Comment:
for bind to lambda, could you use the concrete type, but not `auto` for parameters?